samples/machine-learning/wine/Training.qs

```diff
@@ -54,8 +54,8 @@ namespace Microsoft.Quantum.Samples {
                 w/ LearningRate <- 0.4
                 w/ MinibatchSize <- 2
                 w/ Tolerance <- 0.01
-                w/ NMeasurements <- 10000
-                w/ MaxEpochs <- 16
+                w/ NMeasurements <- 100000
+                w/ MaxEpochs <- 10
```

Comment:
Sorry, I think I missed a discussion somewhere, but what was the reason for the change?